Makefile

```diff
@@ -1,157 +1,600 @@
-		../../../../Qt/4.8.5/mkspecs/features/win32/windows.prf \
-		../../../../Qt/4.8.5/mkspecs/features/resources.prf \
-		../../../../Qt/4.8.5/mkspecs/features/uic.prf \
-		../../../../Qt/4.8.5/mkspecs/features/yacc.prf \
-		../../../../Qt/4.8.5/mkspecs/features/lex.prf \
-		c:/Qt/4.8.5/lib/qtmaind.prl
-	$(QMAKE) -o Makefile dimecoin-qt.pro
-..\..\..\..\Qt\4.8.5\mkspecs\features\device_config.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\qconfig.pri:
-..\..\..\..\Qt\4.8.5\mkspecs\modules\qt_webkit_version.pri:
-..\..\..\..\Qt\4.8.5\mkspecs\features\qt_functions.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\qt_config.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\win32-g++\qmake.conf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\exclusive_builds.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\default_pre.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\default_pre.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\debug.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\debug_and_release.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\default_post.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\default_post.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\static.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\thread.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\rtti.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\exceptions.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\stl.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\warn_on.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\qt.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\moc.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\win32\windows.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\resources.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\uic.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\yacc.prf:
-..\..\..\..\Qt\4.8.5\mkspecs\features\lex.prf:
-c:\Qt\4.8.5\lib\qtmaind.prl:
-qmake: qmake_all FORCE
-	@$(QMAKE) -o Makefile dimecoin-qt.pro
-
-qmake_all: FORCE
-
-make_default: debug-make_default release-make_default FORCE
-make_first: debug-make_first release-make_first FORCE
-all: debug-all release-all FORCE
-clean: debug-clean release-clean FORCE
-	-$(DEL_FILE) C:/Users/simone/GitHub/dimecoin/src/leveldb/libleveldb.a;
-	-$(DEL_FILE) cd
-	-$(DEL_FILE) C:/Users/simone/GitHub/dimecoin/src/leveldb
-	-$(DEL_FILE) ;
-	-$(DEL_FILE) $(MAKE)
-	-$(DEL_FILE) clean
-distclean: debug-distclean release-distclean FORCE
-	-$(DEL_FILE) Makefile
-
-C:/Users/simone/GitHub/dimecoin/src/leveldb/libleveldb.a: FORCE
-
-check: first
-
-debug-mocclean: $(MAKEFILE).Debug
-	$(MAKE) -f $(MAKEFILE).Debug mocclean
-release-mocclean: $(MAKEFILE).Release
-	$(MAKE) -f $(MAKEFILE).Release mocclean
-mocclean: debug-mocclean release-mocclean
-
-debug-mocables: $(MAKEFILE).Debug
-	$(MAKE) -f $(MAKEFILE).Debug mocables
-release-mocables: $(MAKEFILE).Release
-	$(MAKE) -f $(MAKEFILE).Release mocables
-mocables: debug-mocables release-mocables
-FORCE:
-
-$(MAKEFILE).Debug: Makefile
-$(MAKEFILE).Release: Makefile
+#############################################################################

```

Comment:
Why did these make files change so dramatically? I don't want to go through them line by line in the diff, could you give me a tl;dr for the update? I see a lot of Windows file system references, will this build in linux?

Reply:
@CryptoAxl said -

The changes in makefile are not relevant as these are automatically generated by qmake. So these file will always have fixed path depending on where compilation was tested the last time. Actually makefile can remain unchanged. It's the .pro which matters. Previous makesfiles had also specific path cause they were generated by previous compilations Sources are compiled on both linux and Windows. Last compile i did was on windows, that's why.

Reply:
Ah, okay. That makes sense. Let's remove the makefile's from the repo and add to gitignore then. Don't want auto generated files in source control.